test/unit/common/models/PullRequest_test.js

```diff
@@ -85,5 +85,21 @@ describe('Unit | Common | Models | PullRequest', function () {
       // then
       expect(createdInstance.toString()).to.equal(expectedString);
     });
+
+    it('should return a string with properties and end with a dot if it was not there', function () {
```

Comment:
Remarque : Le test précédent vérifie déjà qu'on a les bonnes propriétés, on pourrait se contenter uniquement de vérifier qu'il y a un point dans la string finale, et au passage modifier le titre du test

```suggestion
    it('should return a string which ends with a dot if it was not there', function () {
```